configs/10.0/packages/python/sources

```diff
@@ -41,24 +41,17 @@ ATSRC_PACKAGE_BUNDLE=toolchain_extra
 atsrc_get_patches ()
 {
 	at_get_patch \
-		https://raw.githubusercontent.com/powertechpreview/powertechpreview/26747dc0bfb0f8f30de9ebad78847a96d3191ba5/Python%20Fixes/python-3.5.1-getlib64s1.patch \
-		15ffa7f1c39763b7bf3cca0fd70bf421 || return ${?}
-
+		https://raw.githubusercontent.com/powertechpreview/powertechpreview/master/Python%20Fixes/python-3.5.3-getlib64s1.patch \
```

Comment:
It' s better to get the patch from the commit ID instead of from master, this way you can guarantee its integrity.
If, for example, we remove the patch an old build will continue to work.